app/flatpak-builtins-add-remote.c

```diff
@@ -361,8 +362,13 @@ flatpak_builtin_add_remote (int argc, char **argv,
     return FALSE;
 
   /* We can't retrieve the extra metadata until the remote has been added locally, since
-     ostree_repo_remote_fetch_summary() works with the repository's name, not its URL. */
-  return flatpak_dir_update_remote_configuration (dir, remote_name, cancellable, error);
+     ostree_repo_remote_fetch_summary() works with the repository's name, not its URL.
+     Don't propagate errors here because the remote should be usable already. We
+     might just be offline. */
+  if (!flatpak_dir_update_remote_configuration (dir, remote_name, cancellable, &local_error))
+    g_printerr (_("Warning: could not update extra metadata for '%s': %s\n"), remote_name, local_error->message);
```

Comment:
Fixed, switched to g_warning and had a dig around - looks like G_IO_ERROR_FAILED is the best we can hope to get from ostree-fetcher-*/ostree-pull. It's not precise enough to not print a warning I don't think?